website/src/css/tsp.css

```diff
@@ -2,12 +2,6 @@
 :root[data-theme="light"],
 :root[data-theme="dark"] {
   --header-height: 50px;
-  --sl-color-text-accent: var(--colorBrandForeground1);
-  --sl-color-gray-5: var(--colorNeutralStroke1);
-}
-
-:root[data-theme="dark"] {
-  --sl-color-bg-inline-code: var(--ec-frm-edBg);
```

Comment:
This wasn't specifically flagged but I don't think the a18y automated checks flag dark-mode issues. We removed this override in typespec-azure and doing so does improve the contrast of code blocks in dark mode - so removing here to match.